reconfiguration/cmf/concord.cmf

```diff
@@ -264,6 +264,7 @@ Msg GetDbCheckpointInfoRequest 59 {
 
 Msg GetDbCheckpointInfoStatusResponse 62 {
     list DbCheckpointInfo db_checkpoint_info
+    string additional_info
```

Comment:
nit: can we use a readable name in place of additional_info like available disk space or anything?

Reply:
kept name generic. "available disk space" is actually the information it gives